processing-service-container/pom.xml

```diff
@@ -327,7 +327,7 @@
                             <alias>processing-service</alias>
                             <name>${caf.container.processing.service.name}</name>
                             <build>
-                                <from>node:4.4.7</from>
+                                <from>cafinternal/prereleases:opensuse-nodejs-1.0.0-SNAPSHOT</from>
```

Comment:
As discussed, let's make this `nodejs4`.

Reply:
Done, dev build is still passing